modules/standard/Path.chpl

```diff
@@ -610,6 +624,95 @@ proc file.realPath(out error: syserr): string {
   return "";
 }
 
+// Compute the common prefix length between two lists of path components.
```

Comment:
nit: Still good to use doc-string `/* ... */` comments even for non-chpldoc comments.